docs/workflows/outputs/google-pub-sub.md

```diff
@@ -1,8 +1,8 @@
-# Google Pub/Sub
+# GCP Pub/Sub
```

Comment:
Make the same changes you made for this file in the "Google Pub/Sub" trigger as well. (docs/workflows/triggers/google-pub-sub.md and the docs navigation)